docs/organizations/verified-domains.mdx

```diff
@@ -1,38 +1,57 @@
 ---
-title: Verified Domains
+title: Verified domains
 description: Learn about how to use verified domains for organizations.
 ---
 
-# Creating and verifying a domain
+# Verified domains
```

Comment:
❓ Should we add more info about the restrictions that are in place for which email domains can be used? In particular, we don't allow disposable domains or common email providers (e.g. you cannot create a verified domain for `gmail.com`)